network/player.go

```diff
@@ -17,13 +17,13 @@ func newPlayersFromModels(pms []model.Player) []Player {
 	return ps
 }
 
-func NewGetPlayerResponseFromModel(pm model.Player) GetPlayerResponse {
+func ConvertToGetPlayerResponse(p model.Player) GetPlayerResponse {
 	return GetPlayerResponse{
 		Player: Player{
-			ID:   pm.ID,
-			Name: pm.Name,
+			ID:   p.ID,
+			Name: p.Name,
 		},
-		Games: convertGames(pm.Games),
+		Games: convertGames(p.Games),
```

Comment:
As I consider this; perhaps we do not want to return _all_ games on the "get player response". This allows the "get player request" to grow unbounded. We'll likely end up wanting a "get active games", "get completed games", "get games against opponent x", "get games with partner y", in the future, but for now this should be fine. I filed https://github.com/joshprzybyszewski/cribbage/issues/60 to address that work